includes/Admin/TinyMCE.php

```diff
@@ -118,11 +119,12 @@ public function tinymce_web_stories_button( array $buttons ): array {
 	 *
 	 * @since 1.5.0
 	 *
-	 * @param array $plugins Array of TinyMCE plugin scripts.
+	 * @param array|mixed $plugins Array of TinyMCE plugin scripts.
 	 *
 	 * @return array
 	 */
-	public function web_stories_mce_plugin( array $plugins ): array {
+	public function web_stories_mce_plugin( $plugins ) {
+		$plugins                = (array) $plugins;
```

Comment:
Here you cast to an array, but not in other places like `action_links()`. Any specific reason?

Reply:
Nope, I just missed them. 😄